### ISSUE TITLE
Duplicate search tokens 

### ISSUE THREAD

**Issue by Dominik-O22** [thirdy/durian#62]
Because, the config file adds "tmpsc bo online"  to every search it is impossible without changing the config file to search indifferent leagues, or for items without a buyout or offline items. I think if possible they should only be added if no such identifier is given instead of them always being added

**Comment by kyri0** [comment]
Something like this was already requested somewhere, I believe in the forums. Thirdy said he would look for a way to make the manual input of bo, league or online to override the input from the config file. Glad you made a ticket for it!

About the league, I don't know if you can use xyzz indexer to search more than one league in one single request, desplaying like standard and tmpstandard results at the same time. At least, in the front end that xyzz provides, which is poe.trade, I don't know how to do this. Can you do this there?

**Comment by Dominik-O22** [comment]
I think i didnt express clearly enough what i meant, i dont want to be able to search in multiple leagues at once, it was part of the list of things you can't do without changing the config

**Comment by thirdy** [comment]
how about leaving the config empty?

```
manual.search.prefix=
automated.search.prefix=
```

?

**Comment by Dominik-O22** [comment]
yes that would work problem is the lose of convenience they provide

**Comment by thirdy** [comment]
i see, some kind of override to the defaults would be good for this.

moving the parsing of search terms into js script might be the way to go for this
